docs/montecarlo/sourceintegration.rst

```diff
@@ -0,0 +1,112 @@
+********************************
+Direct integration of the radiation field
+********************************
+
+.. note::
+
+  The current implementation only works with the downbranch line interaction scheme.
+
+
+:cite:`Lucy1999a` describes an alternative method for the generation of spectram for the supernova ejecta. Instead of using the frequency and energy of virtual montecarlo packets to create a spectrum through binning, one can use estimators collected during the montecarlo simulation to formally integrate the radiation field. Spectra generated using this method do not contain montecarlo noise directly. Here the monte carlo nature of the simulation only affects the strengths of lines and thus the spectra appear to be of better quality.
+
+The procedure uses a line absorption rate estimator that is collected during the montecarlo simulation:
+
+.. math::
+
+   \dot E_{lu} = \frac{1}{\Delta t V} \left( 1- e^{-\tau_{lu}}\right) \sum \epsilon
+
+where the sum is over all the packages in a given shell that come into resonance with the transition :math:`u \rightarrow l` during the monte carlo run, :math:`\epsilon` is the energy of one such packet, and :math:`\tau_{lu}` the optical depth of the line.
+
+After the final monte carlo step, a level absorption estimator is calculated, given by:
+
+.. math::
+
+   \dot E_u = \sum_{i < u} \dot E_{iu}
+
+that is, by summing all the line absorption estimators below the currently selected level.
+
+The source function for each line can then be derived from the relation
+
+.. math::
+
+   \left( 1- e^{-\tau_{lu}}\right) S_{ul} = \frac{\lambda_{ul} t}{4 \pi} q_{ul} \dot E_u
+
+where :math:`\lambda_{ul}` is the wavelength of each line  :math:`u \rightarrow l`, and :math:`q_{ul}` is the corresponding branching ratio. The attenuating factor is kept on the left hand side because it is the product of the two that will appear in later formulae.
+
+Having thus produced attenuated source functions from our Monte Carlo run, we move on to using these to calculate the emerging intensity and finally the luminosity per wavelength. The final integral is given as
```

Comment:
The formal integration is based on the so-called "elementary supernova" model, which is described in detail in Jeffery & Branch 1990.